openapi/components/schemas/PaymentTokens/PaymentInstruments/BBANInstrument.yaml

```
@@ -1,35 +1,35 @@
 description: Bank account BBAN instrument.
 allOf:
   - $ref: ./BankAccountInstrument.yaml
-  - type: object
-    required:
-      - accountNumber
-      - routingNumber
-      - accountType
-    properties:
-      accountNumber:
-        description: Bank Account Number.
-        type: string
-        pattern: '^[0-9]+$'
-        writeOnly: true
-      routingNumber:
-        description: Bank Routing Number.
-        type: string
-        pattern: '^[0-9]+$'
-      accountType:
-        description: Bank Account Type.
-        type: string
-        enum:
-          - checking
-          - savings
-          - other
-      bic:
-        description: Bank Identifier Code.
-        type: string
-      bankName:
-        description: Bank name.
-        type: string
-      last4:
-        description: Bank Account Number's last 4 digits.
-        type: string
-        readOnly: true
+type: object
+required:
+  - accountNumber
+  - routingNumber
+  - accountType
+properties:
+  accountNumber:
+    description: Bank Account Number.
```

nitpick: we should use sentence case here, unless we are describing an acronym 
```suggestion
    description: Customer's bank account number.
```
or 
```suggestion
    description: Customer's Bank Account Number (BAN).
```